docs/publishing/index.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Publishing Web Components
+section: guides
+tags:
+  - guides
+  - guide
+---
+
+## Publishing Web Components
+
+If you have developed a package that you would like to make available to other developers via a public registry, take the following concepts into account to ensure that external users of your code may make the most of it in their projects. If you've made it this far, your package will likely already include things like [demos](/demoing/), [documentation](/demoing/storybook-addon-markdown-docs.html), [tests](/testing/), et al. Each of these play a role in making the clearest example of the benefits on your work, outlining the easiest path to integrating that work into other projects, and ensuring that the package as a whole is resilient to change over time, but are not discussed directly below. If you're looking for information on including these things in your published packages, take a look at our [guides](/guide/index.html) or learn about [getting started](#getting-started) below.
+
+Keep in mind that these suggestions are not specifically tuned for publishing to a CDN or other contexts where broader application-specific optimizations will not be applied to the code available therein.
+
+<style>
+.content__default ul {list-style:none;padding:0;}
+</style>
+
+### Do: 👍
+
+- ✅ [Do publish latest standard EcmaScript](#do-publish-latest-standard-ecmascript)
+- ✅ [Do publish standard es modules](#do-publish-standard-es-modules)
+- ✅ [Do include appropriate `package.json` entries](#do-include-appropriate-packagejson-entries)
```

Comment:
for lazy people like me this list should have all the info needed to publish something without reading everything in detail... 

e.g. include the appropriate package.json entries in this list